storybookjs/storybook
# Clearer descriptions for state addons

removed duplicated addons 

Issue:

## What I did
I removed duplicated addons

## How to test
There are two of them 

Is this testable with Jest or Chromatic screenshots? no
Does this need a new example in the kitchen sink apps? no
Does this need an update to the documentation? yes

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
